Funcky/Extensions/DictionaryExtensions.cs

```diff
@@ -15,6 +17,7 @@ public static Option<TValue> GetValueOrNone<TKey, TValue>(this IDictionary<TKey,
             : Option<TValue>.None;
 
     [Pure]
+    [OverloadResolutionPriority(10)]
```

Comment:
10 looks pretty arbitrary. The examples are usually all with 1,2,3.

Reply:
Maybe add a test in DictionaryExtensionTest or, remove the hint in one of the tests.

Reply:
Fixed in [ba75ecb](https://github.com/polyadic/funcky/pull/798/commits/ba75ecbdbfbcfc9cbaac9763305ad21113dfa910), there is already a test for this in `Funcky.Test/Extensions/DictionaryExtensionTest.cs`.